src/Login.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+class Login extends React.Component {
+  render() {
+    return (
+      <form>
+        <div className="three columns">
+          <label htmlFor="Email">Email Address</label>
+          <input type="email" placeholder="Email Address" />
+        </div>
+        <div className="three columns">
+          <label htmlFor="Username">Password</label>
```

Comment:
<label htmlFor="Username">Password</label>
should be
<label htmlFor="Password">Password</label>